File: APLICACION_MOVIL/lib/domain/entities/student.dart
class Student {
  String _id;
  String _name;
  String _pictureURL;

  Student(this._id, this._name, this._pictureURL);

  String get id => _id;
  String get name => _name;
  String get pictureURL => _pictureURL;

  set id(String id) => this._id = id;
  set name(String name) => this._name = name;
  set pictureURL(String pictureURL) => this._pictureURL = pictureURL;

  Student.map(dynamic obj) {
    this._id = obj['id'];
    this._name = obj['name'];
    this._pictureURL = obj['picture_url'];
  }
}

File: SERVIDOR SDR_SCR/lib/domain/entities/student.py
class Student:
    def __init__(self, ID: str, name: str, pictureURL: str):
        self.__ID = ID
        self.__name = name
        self.__pictureURL = pictureURL

    def __str__(self) -> str:
        return f'({self.__ID}, {self.__name}, {self.__pictureURL})'

    def __eq__(self, other):
        return ((self.__ID == other.__ID) and (self.__name == other.__name) and (self.__pictureURL == other.__pictureURL))

    def __hash__(self):
        return hash(f'{self.__ID}{self.__name}{self.__pictureURL}')

    def getID(self) -> str:
        return self.__ID

    def getName(self) -> str:
        return self.__name

    def getPictureURL(self) -> str:
        return self.__pictureURL

    def toJSON(self) -> dict:
        jsonObject = {
            'id': self.__ID,
            'name': self.__name,
            'picture_url': self.__pictureURL
        }

        return jsonObject